packages/conferer/src/Conferer/Config/Internal/Types.hs

```diff
@@ -29,7 +29,13 @@ data Config =
   { configSources :: [Source]
   , configDefaults :: Map Key [Dynamic]
   , configKeyMappings :: [(Key, Key)]
-  } deriving (Show)
+  }
+
+instance Show Config where
+  show _ = "Config"
+
+instance Eq Config where
+  _ == _ = True
```

Comment:
So this is a hack, the point of this is that many of the new exceptions have the whole `Config`, so that means that everywhere else I try to compare something that has a `Config` I have to deal with an `Eq` instance that I really don't care about.

Alternatives are:
* implementing `Eq` and `Show` manually for everything that contains a `Config`
* Adding some kind of random id that gets populated when the `Config` is instantiated but I felt like I was overengineering